scripts/dts/edtlib.py

```diff
@@ -175,13 +176,19 @@ def __init__(self, dts, bindings_dirs, warn_file=None,
         default_prop_types (default: True):
           If True, default property types will be used when a node has no
           bindings.
+
+        support_fixed_partitions_on_any_bus (default True):
+          If True, set the Node.bus for 'fixed-partitions' compatible nodes
+          to None.  This allows 'fixed-partitions' to bind if they are under a SPI
+          controller bus or similar case.
         """
         # Do this indirection with None in case sys.stderr is deliberately
         # overridden
         self._warn_file = sys.stderr if warn_file is None else warn_file
 
         self._warn_reg_unit_address_mismatch = warn_reg_unit_address_mismatch
         self._default_prop_types = default_prop_types
+        self._fixed_partions_no_bus = support_fixed_partitions_on_any_bus
```

Comment:
Please rename the property to either `_fixed_partitions_no_bus` or `_fixed_partitions_on_bus`, whichever is correct.  Either way `partions` isn't right.

Reply:
Fixed.